dictionaries/pt/filters.yml

```diff
@@ -1,31 +1,31 @@
 pt:
-  edit_sumamry: 'relatórios %{num} filtros editar obsoletos'
+  edit_summary: 'Informando %{num} filtros de edição obsoletos'
   last_changed: 'Última [[Especial:Filtro de abusos/history/%{id}|alteração]]'
-  summary: '%{num} filtros editar ativado sem visitas nos últimos %{days} dias'
-  title: 'Filtros editar obsoletos como de %{date}'
-  'abuseFilter': 'Filtro de abusos'
+  summary: '%{num} filtros de edição ativados sem ocorrências nos últimos %{days} dias'
+  title: 'Filtros de edição obsoletos em %{date}'
+  'abuseFilter': 'Filtro de edições'
   'actions': 'Ações'
   'at': 'em'
   'by': 'por'
   'description': 'Descrição'
   'disallow': 'não autorizar'
-  'edit filters': 'filtros editar'
-  'failed to write to template': 'Falha ao gravar no modelo'
+  'edit filters': 'filtros de edição'
+  'failed to write to template': 'Falha ao gravar na predefinição'
   'fatal error': 'Erro fatal'
   'filter': 'Filtro'
-  'enabled filters with no hits in %{num}': 'filtros ativados sem visitas em'
+  'enabled filters with no hits in %{num}': 'filtros ativados sem ocorrências em'
   'flags': 'Sinalizações'
```

Comment:
@he7d3r Flags I guess are something I made up. They include `enabled`/`disabled` and `private`/`public`. So "flags" in the sense of computer science (e.g. booleans). Maybe I should break them up into `status` and `visibility` to match Special:AbuseFilter, it's just nice to consolidate in the report so it's not a really long line of text. Or maybe there's a better term we could use?